### DISCUSSION TITLE
Flowise - OpenAI Assistant cost tracking

### DISCUSSION THREAD

**Discussion by jozsef-gereby** [langfuse/langfuse#969] [Ideas]
In Langfuse the cost of an OpenAI Assistant doesn't show up from Flowise. I see on the Flowise debug console that the token counts are present. `usage: { prompt_tokens: 6285, completion_tokens: 162, total_tokens: 6447 }`
Perhaps this would be sufficient and useful data to pass forward between Flowise and Langfuse to calculate the run costs.

**Comment by daffaalex22** [comment]
Bumping this up
